hw_07/AssemblyTwo/Program.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace AssemblyTwo {
+    class Program {
+        static void Main(string[] args) {
+            Console.WriteLine("Hello World!");
```

Comment:
the same

here you need to demonstrate all possible modifiers are set up in Motorcycle, Chopper types